lib/evmone/instructions.hpp

```diff
@@ -764,6 +764,24 @@ inline Result gas(StackTop stack, int64_t gas_left, ExecutionState& /*state*/) n
     return {EVMC_SUCCESS, gas_left};
 }
 
+inline void tload(StackTop stack, ExecutionState& state) noexcept
+{
+    const auto key = intx::be::store<evmc::bytes32>(stack.pop());
+    const auto value = state.host.get_transient_storage(state.msg->recipient, key);
+    stack.push(intx::be::load<uint256>(value));
+}
+
+inline Result tstore(StackTop stack, int64_t gas_left, ExecutionState& state) noexcept
+{
+    if (state.in_static_mode())
+        return {EVMC_STATIC_MODE_VIOLATION, 0};
```

Comment:
Would be good to have this like covered with unit test.

Reply:
Added `transient_storage_static` test.